eks/cluster-loader/cluster-loader.gotmpl

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cl-service
```

Comment:
I'd just call these by the name of the application that uses them. "clusterloader2". It's of type service account, so saying its the cl-service service account stutters. This is opposed to the "clusterloader2 service account".

Reply:
+1